src/main/java/Duke.java

```diff
@@ -1,10 +1,297 @@
+        }
+        else{
+            printInvalidTaskWarning(jobNumber);
+        }
+    }
+
+    private static void runList(Task[] taskList) {
+        int numbering = 1;
+
+        // error handling - no jobs
+        if(Task.taskCount == 0){
+            System.out.println("No tasks yet! What would you like to do today?\n");
+        }
+        else {
+            for (int i = 0; i < Task.taskCount; i++) {
+                System.out.print(numbering + ". ");
+                taskList[i].printTask();
+                numbering++;
+            }
+            System.out.println();
+        }
+    }
+
+    private static void runTodo(Task[] taskList, String input) {
+
+        if(input == null){
+            printInvalidInputWarning();
+            return;
+        }
+
+        Todo newTask = new Todo(input);
+
+        taskList[Task.taskCount] = newTask;
+        Task.taskCount++;
+
+        printTaskAdded(newTask);
+    }
+
+    private static void runDeadline(Task[] taskList, String input, String by){
+
+        if(input == null || by == null){
+            printInvalidInputWarning();
+            return;
+        }
+
+        Deadline newTask = new Deadline(input, by);
+        taskList[Task.taskCount] = newTask;
+        Task.taskCount++;
+
+        printTaskAdded(newTask);
+
+    }
+
+    private static void runEvent(Task[] taskList, String input, String by){
+
+        if(input == null || by == null){
+            printInvalidInputWarning();
+            return;
+        }
+
+        Event newTask = new Event(input, by);
+        taskList[Task.taskCount] = newTask;
+        Task.taskCount++;
+
+        printTaskAdded(newTask);
+
+    }
+
+    /**
+     * IDK TO CLASSIFY
+     * TODO
+     * */
+
+    private static String parseJob(String input, String type) {
+
+        String[] words = input.split(" ");
+
+        if(words.length < 2){
+            return null;
+        }
+
+        return getJobString(words, type);
+    }
+
+    private static String parseDate(String input, String type) {
+
+        String[] words = input.split(type);
+
+        if(words.length == 1){
+            return null;
+        }
+
+        return words[1].trim();
+    }
+
+    private static String getJobString(String[] words, String type) {
+
+        String job = words[1];
+
+        for(int i = 2; i< words.length; i++){
+            if(words[i].equalsIgnoreCase(type)){
+                break;
+            }
+            job += " " + words[i];
+        }
+        return job;
+    }
+
+    private static boolean startsWith(String input, String command){
+        return input.toUpperCase().startsWith(command.toUpperCase());
+    }
+
+    private static void markJobAsDone(Task task) {
+        task.setDone(true);
+        System.out.print("Congrats! You've completed: \n   ");
+        task.printTask();
+        System.out.println();
+    }
+
+
+
+    /**
+     * PRINTING METHODS
+     * */
+
+    private static void printTaskAdded(Task task) {
```

Comment:
Overall, I like that you have camelCased your method and variable names consistently.